package.json

```diff
@@ -113,6 +113,7 @@
     "minimist": "^1.2.0",
     "moment": "^2.13.0",
     "node-fetch": "^1.6.0",
+    "omit-empty": "^0.4.1",
```

Comment:
Can't we use `_.omit` from `lodash` for that? This we we'd save on external dependency.

Reply:
Unfortunately, from my experience `_.omit` does not work for deep values.